src/Numerics/Exceptions.cs

```diff
@@ -20,7 +20,7 @@ public NonConvergenceException(string message) : base(message)
         public NonConvergenceException(string message, Exception innerException) : base(message, innerException)
         {
         }
-#if !PORTABLE
+#if !PORTABLE && !NOSERIALIZATION
```

Comment:
Just for the record, the symbols we currently use are documented [here](https://github.com/mathnet/mathnet-numerics/blob/master/docs/content/Compatibility.md) (rendered [here](https://numerics.mathdotnet.com/Compatibility.html#Platform-Discrepancies))